alf/environments/suite_gym.py

```diff
@@ -90,8 +90,8 @@ def wrap_env(gym_env,
     transpose, 'image_channel_first' is set as True by default. There are two options
     provided in ALF to handle this transpose:
         1. Applying the gym_wrappers.ImageChannelFirst after all gym_env_wrappers
-            and before the TorchGymWrapper.
-        2. Applying the torch_wrappers.ImageChannelFirst after all torch_gym_wrappers.
+            and before the AlfGymWrapper.
+        2. Applying the alf_wrappers.ImageChannelFirst after all alf_gym_wrappers.
     The first option is used in current function.
```

Comment:
The second option becomes meaningless now given that numpy array is used before and after AlfGymWrapper, so maybe only keep the following:

... 'image_channel_first' is set as True by default. gym_wrappers.ImageChannelFirst is applied after all gym_env_wrappers and before the AlfGymWrapper.

Reply:
updated